docs/skopeo-standalone-verify.1.md

```diff
@@ -8,7 +8,8 @@ skopeo\-standalone\-verify - Verify an image signature.
 
 ## DESCRIPTION
 
-Verify a signature using local files, digest will be printed on success.
+Verify a signature using local files, digest will be printed on success. This is primarily a debugging tool, or useful for special cases,
+and usually should not be a part of your normal operational workflow.
```

Comment:
… ; configure a policy.json (see containers-policy.json(5)) instead.

Reply:
Agreed, and have changed - thanks for the kind feedback 👍